extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialogModel.ts

```diff
@@ -70,6 +70,18 @@ export class MigrationCutoverDialogModel {
 		return undefined!;
 	}
 
+	public async fetchErrors() {
+		let errors = [];
+		await this.fetchStatus();
+		errors.push(this.migrationOpStatus.error?.message);
+		errors.push(this._migration.asyncOperationResult?.error?.message);
+		errors.push(this.migrationStatus.properties.provisioningError);
+		errors.push(this.migrationStatus.properties.migrationFailureError?.message);
+		errors.push(this.migrationStatus.properties.migrationStatusDetails?.fileUploadBlockingErrors ?? []);
```

Comment:
You seem to have a mix of strings and arrays of strings in the returned errors. It's generally better to stick to one or the other for ease of use by callers